scripts/updatedep.sh

```diff
@@ -36,6 +36,10 @@ pushd "${GLIDE_ROOT}"
 	go install
 popd
 
+echo "vendoring gofail/runtime"
```

Comment:
is this necessary? the updatedep script probably shouldn't have special rules for any deps